tests/integration_tests/performance/test_block_performance.py

```diff
@@ -66,6 +66,8 @@ def run_fio(env_id, basevm, mode, bs):
     """Run a fio test in the specified mode with block size bs."""
     logs_path = f"{basevm.jailer.chroot_base_with_id()}/{env_id}/{mode}{bs}"
 
+    num_jobs = CONFIG["load_factor"] * basevm.vcpus_count
```

Comment:
could be helpful to add a comment why `load_factor` is a part of this calculation

Reply:
I would be, but I also don't know much more behind its intention beyond what the name would imply :sweat_smile:

Reply:
I think even this is a valuable piece of info ;)

Reply:
Well, the comment would be "load factor determines how many workers to spawn per vcpu", which only raises more questions: Why would we want to have more than 1 worker per vcpu? What's the impact of having it not be 1? I'll just... remove the variable I think :joy:

Reply:
I was only able to grep this variable set to 1 in the code, which is currently a no-op. I would ideally look for a comment explaining why we believe it's a part of the equation and why it is going to hold when/if we introduce other values than 1.

Reply:
It's been there (and set to 1) since the beginning of time, with no explanation.